hypervisor/boot/reloc.c

```diff
@@ -59,7 +59,9 @@ void relocate(void)
 {
 #ifdef CONFIG_RELOC
 	struct Elf64_Dyn *dyn;
-	struct Elf64_Rel *start = NULL, *end = NULL;
+	struct Elf64_Rel *entry = NULL;
+        uint8_t *rela_start = NULL, *rela_end = NULL;
```

Comment:
Are u using TAB or SPACE?

Reply:
There are some spaces in this patch. Need to change to tabs.

Reply:
Yes, there are some white spaces introduced inadvertently, I will fix it, thanks you @dongyaozu and @ZideChen0 point it out.